### DISCUSSION TITLE
Role options are not being shown

### DISCUSSION THREAD

**Discussion by marcelomirandasilva** [nocodb/nocodb#7061] [Q&A] [answered]
Differently from what the documentation says, I can only see the "Creator" and "Viewer" options, can anyone help me figure out how to show the other user options?

![image](https://github.com/nocodb/nocodb/assets/13606710/2be76589-f8e9-4d60-bb65-9539e5b9c4b9)

**Comment by mertmit** [comment, marked as answer]
Those are organization level roles (org level creators can create their own bases whereas org level viewers can only access bases they are invited, they can have any base role (creator, editor, commenter, viewer) regardless of their org role), after inviting an user you can assign base role from Members page inside your base:
<img width="1800" alt="image" src="https://github.com/nocodb/nocodb/assets/59797957/940e2b59-ea17-4330-8a5a-1b2bad0c0f99">

**Comment by marcelomirandasilva** [comment]
thankyou
